eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -219,15 +160,41 @@ jobs:
         - pwsh: npm install -g moxygen
           displayName: Install Moxygen to generate markdown for docs.microsoft.com
 
+        - pwsh: azcopy copy "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp/0.15.0/*" "$(System.DefaultWorkingDirectory)/parser" --recursive
```

Comment:
Can you use a variable to set this version and use variable instead of version number directly here?